backend/controllers/vendorControllers.js

```
@@ -0,0 +1,230 @@
+  return null;
+};
+
+const validateInputFormats = (email, mobile) => {
+  const emailValid = !email || /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/.test(email);
+  const mobileValid = !mobile || /^\+?[0-9]{10,15}$/.test(mobile);
+  return { emailValid, mobileValid };
+};
+
+const handleImageDeletion = (existingImages, removedImages) => {
+  if (!removedImages) return existingImages;
+  if (!Array.isArray(removedImages)) removedImages = [removedImages];
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)